cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -99,6 +99,41 @@ const char *log_level(long level)
 #define LOG_SIZE    512
 static char log_buff[LOG_SIZE];
 
+
+/** Log as a backup, if python isn't working */
```

Comment:
Can you be more descriptive when this function will be used? What conditions make Python not working?